README.md

```diff
@@ -3,6 +3,18 @@ An example bot implemented in JavaScript
 
 If you are looking for the java example bot please go [here](https://github.com/RLBot/RLBotJavaExample)
 
+## Installation guide
+
+If you've installed a few other npm packages already, chances are doing `npm install SuperVK/RLBotJS` works.
+
+### if `npm install SuperVK/RLBotJS` gives an error follow this guide:
+
+1. Install python 2.7, but make sure to not check "add to PATH" in the installation, because then the python runner for RLBot won't run anymore
```

Comment:
It seems like these installation steps would be required for anybody who just wanted to run the bot, e.g. tournament hosts, bot pack users. If that's the case, this would be our worst setup experience, and I probably wouldn't recommend JS bots to people who join discord looking for a challenge. Could we use something like pkg, maybe? https://www.npmjs.com/package/pkg

As it stands, people will probably run into a lot of problems and ask a lot of questions, which would be painful for you, me, and everybody else who helps with troubleshooting. We should try hard to make it easier, and if we can't, we'll have to make a hard decision about whether JS can be officially supported.

Reply:
IamEld3st has talked about packaging python 2.7 (and even nodejs itself I think?) into the new GUI, so for playing with the bot/running them in tournament will be simple. (Ask him for info on that)

But for the developers, we will have to think about it. It's not a problem this package is alone in, every node package that uses C++ (we depend on 2 packages that do) must be compiled using these steps (only on Windows). So people who have been using various packages will have set this up, and will be able to run `npm install rlbot` without too much trouble. (I asked a friend if he could install the package, he had no problems, and didn't need to use the guide, but someone in the rlbot discord server who wanted to try it out had a lot more issues)